HeterogeneousTest/ROCmOpaque/test/testROCmTestAdditionModules.py

```diff
@@ -0,0 +1,31 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process('TestROCmTestOpaqueAdditionModule')
+
+process.source = cms.Source('EmptySource')
+
+process.ROCmService = cms.Service('ROCmService')
+
+process.rocmTestDeviceAdditionModule = cms.EDAnalyzer('ROCmTestDeviceAdditionModule',
+    size = cms.uint32( 1024*1024 )
+)
+
+process.rocmTestKernelAdditionModule = cms.EDAnalyzer('ROCmTestKernelAdditionModule',
+    size = cms.uint32( 1024*1024 )
+)
+
+process.rocmTestWrapperAdditionModule = cms.EDAnalyzer('ROCmTestWrapperAdditionModule',
+    size = cms.uint32( 1024*1024 )
+)
+
+process.rocmTestOpaqueAdditionModule = cms.EDAnalyzer('ROCmTestOpaqueAdditionModule',
+    size = cms.uint32( 1024*1024 )
+)
+
+process.path = cms.Path(
+     process.rocmTestDeviceAdditionModule +
```

Comment:
@makortel note that this one works with ROCm, while it does not work with CUDA.

Reply:
Nice